src/pages/registration/registration.tmpl.js

```diff
@@ -0,0 +1,67 @@
+import Handlebars from 'handlebars';
+import field from '../../components/field';
+import button from '../../components/button';
+import tmpl from './registration.hbs';
+
+import './registration.css';
+
+const fields = {
+    firstName: {
+        name: 'firstName',
+        placeholder: 'Имя',
+        clazz: 'registration__first-name'
```

Comment:
```suggestion
        clazz: 'registration__first-name',
```
Лучше ставить тут `,`. Тут можно почитать почему
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas